contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -101,10 +101,16 @@ contract NodeOperatorsRegistry is INodeOperatorsRegistry, IsContract, AragonApp
         validAddress(_rewardAddress)
         returns (uint256 id)
     {
-        id = totalOperatorsCount++;
+        bytes32 totalOperatorsCountPosition = TOTAL_OPERATORS_COUNT_VALUE_POSITION;
+        id = totalOperatorsCountPosition.getStorageUint256();
+        totalOperatorsCountPosition.setStorageUint256(id.add(1));
+
         NodeOperator storage operator = operators[id];
 
-        activeOperatorsCount++;
+        bytes32 activeOperatorsCountPosition = ACTIVE_OPERATORS_COUNT_VALUE_POSITION;
```

Comment:
The same here for `ACTIVE_OPERATORS_COUNT_VALUE_POSITION`.

Reply:
Done